front/app/containers/Admin/projects/project/poll/WrongMaxChoiceIndicator.test.tsx

```diff
@@ -1,117 +1,21 @@
-      );
-      expect(Wrapper).toMatchSnapshot();
-    });
-    it('renders correctly when options is Error', () => {
-      const Wrapper = shallow(
-        <WrongMaxChoiceIndicator
-          questionId="questionId"
-          maxAnswers={5}
-          options={new Error()}
-        />
-      );
-      expect(Wrapper).toMatchSnapshot();
-    });
-    it('renders correctly when options is undefined', () => {
-      const Wrapper = shallow(
-        <WrongMaxChoiceIndicator
-          questionId="questionId"
-          maxAnswers={5}
-          options={undefined}
-        />
-      );
-      expect(Wrapper).toMatchSnapshot();
-    });
-    it('renders correctly when maxAnswers is null', () => {
-      const Wrapper = shallow(
-        <WrongMaxChoiceIndicator
-          questionId="questionId"
-          maxAnswers={null}
-          options={pollOptions}
-        />
-      );
-      expect(Wrapper).toMatchSnapshot();
-    });
-  });
-  describe('renders the error message when max answers is not as expected', () => {
-    it('renders correctly when options is as expected and maxAnswers is zero', () => {
-      const Wrapper = shallow(
-        <WrongMaxChoiceIndicator
-          questionId="questionId"
-          maxAnswers={0}
-          options={pollOptions}
-        />
-      );
-      expect(
-        shallow(Wrapper.find('StyledIconTooltip').prop('content')).prop('id')
-      ).toContain('maxUnderTheMinTooltip');
-      expect(Wrapper.find('FormattedMessage').prop('id')).toContain('wrongMax');
-    });
-    it('renders correctly when options is as expected and maxAnswers is one', () => {
-      const Wrapper = shallow(
-        <WrongMaxChoiceIndicator
-          questionId="questionId"
-          maxAnswers={1}
-          options={pollOptions}
-        />
-      );
-      expect(
-        shallow(Wrapper.find('StyledIconTooltip').prop('content')).prop('id')
-      ).toContain('maxUnderTheMinTooltip');
-      expect(Wrapper.find('FormattedMessage').prop('id')).toContain('wrongMax');
-    });
-    it('renders correctly when there is less options than maxAnswers', () => {
-      const Wrapper = shallow(
-        <WrongMaxChoiceIndicator
-          questionId="questionId"
-          maxAnswers={10}
-          options={pollOptions}
-        />
-      );
-      expect(
-        shallow(Wrapper.find('StyledIconTooltip').prop('content')).prop('id')
-      ).toContain('maxOverTheMaxTooltip');
-      expect(Wrapper.find('FormattedMessage').prop('id')).toContain('wrongMax');
-    });
-  });
-  describe('renders nothing when max answers is as expected', () => {
-    it('renders correclty when 2 <= maxAnswers <= pollOptions.length', () => {
-      const Wrapper = shallow(
-        <WrongMaxChoiceIndicator
-          questionId="questionId"
-          maxAnswers={2}
-          options={pollOptions}
-        />
-      );
-      expect(Wrapper).toMatchSnapshot();
-    });
+  it('renders correctly when there is less options than maxAnswers', () => {
```

Comment:
Should we test the opposite scenario, too?

Reply:
Good suggestion! 👌 I've also tried to improve the test description